### ISSUE TITLE
Add option to auto-start timers

### ISSUE THREAD

**Issue by schmich** [schmich/marinara#11]
- Auto-start focus after break
- Auto-start break after focus

**Comment by schmich** [comment]
Won't fix. Starting focus/break session is easy enough.

**Comment by JohnSpe** [comment]
I would suggest that you kindly rethink this and allow for an optional auto jump to the next period. It would allow to better adhere to the system, to be more disciplined. Since you have to start it yourself you are prone to "yes but first let me..." and there you go breaking the rule. If you had this option that would help to be more strict since it would be out of your control to give you that spare minutes.
Thanks.

**Comment by schmich** [comment]
Fair enough. It's easy enough to implement, I just hadn't heard any requests for it.

Thanks for the feedback.

**Comment by JohnSpe** [comment]
Thank you :)

**Comment by kdragowski** [comment]
Such an option would be very helpful. Often I catch myself that taking a break I forget to run the counter.

**Comment by schmich** [comment]
@kdragowski Thanks for the feedback. I've been meaning to get to this feature for a long time, but I've been sidetracked by other features. Hoping to get to this soon.

**Comment by kdragowski** [comment]
Thank you

**Comment by thaenalpha** [comment]
Yes, I also request for this. It will be a lot helpful. (Come through the chrome extension feedback: latest comment)

**Comment by jh711** [comment]
I wanted to add my voice to those that think a continuous option would be super helpful--for the next focus or break to start automatically after the last focus or break ends, without clicking on the tomato in the Chrome extension. Thanks so much for your work on this!

**Comment by mlevesquedion** [comment]
I would like to have this feature as well. I could look into implementing it if you like.

**Comment by xbootnek** [comment]
I would like this feature too. If possible, and auto start on opening the browser would be great.

**Comment by lreeder14** [comment]
Definitely would like this feature. My focus time ended and when I looked at the time left on my break the timer hadn't started -- I was expecting to to autostart as it is part of the 30 minute cycle. Thanks for considering.

**Comment by Jeanneaple** [comment]
Yes. I have used Marinara for pretty long time now and i love it. I want to suggest adding the options to automatically start the timer in these different moments.
- When we open the browser
- Between the break and working

Many users will prefer to click the button but many will forget to do that.

Thank you for already amazing extension.

**Comment by JeremieLitzler** [comment]
Hi @schmich 

What are we at with this?
Thanks for the awesome extension :)

**Comment by bagustris** [comment]
This is a must-have feature. Here is my rationale. Although it is easy, people tend to FORGET, FORGET, and FORGET. And a machine never forgets. Hence, it is necessary to **automate** the start process, as this app does to automate the timer instead of pressing a timer every 25 seconds. At least, add an option for this feature although it is not in the **default** setting.

I would like to add one request aside from the suggested options above:
- autostart timer when Chrome is opened

**Comment by 04116** [comment]
hi @schmich how about the status of this request? 
Have you used this extension daily?
I am trying to remove this extension just before found this thread
